lib/pure/asyncfutures.nim

```diff
@@ -328,15 +329,16 @@ proc `$`*(stackTraceEntries: seq[StackTraceEntry]): string =
   var indent = 2
   # Format the entries.
   for entry in entries:
-    let (filename, procname) = getFilenameProcname(entry)
+    var (filename, procname) = getFilenameProcname(entry)
 
     if procname == "":
       if entry.line == reraisedFromBegin:
-        result.add(spaces(indent) & "#[\n")
-        indent.inc(2)
+        break
       elif entry.line == reraisedFromEnd:
-        indent.dec(2)
-        result.add(spaces(indent) & "]#\n")
+        doAssert false
+    elif procname.endsWith("Iter"): # generates more unique suffix?
+      procname.setLen(procname.len-4)
+    elif find(nativeToUnixPath(filename), "lib/pure/async") > 0: # is it stdlib?
```

Comment:
We don't skip stacks in the stdlib for other libraries so I don't think we should do it for async either.